### ISSUE TITLE
Force quotes on strings when sorting

### ISSUE THREAD

**Issue by diepes** [pascalre/vscode-yaml-sort#17]
When sorting a yaml block, strings that were double quoted gets stripped of quotes.

Before
```
  - matched_address: "peer1"
    map: "Standard-1"
    seq: "20"
    pfs: "group5"
    peer: "Peer-1"
    ike: "ikev1"
    transform: "Standard-Transform"
    sa: "28800"
```

After

```
  - ike: ikev1
    map: Standard-1
    matched_address: peer1
    peer: Peer-1
    pfs: group5
    sa: '28800'
    seq: '20'
    transform: Standard-Transform
```

**Comment by pascalre** [comment]
Will be working on this, when an upcoming release of js-yaml contains [7b256d7](https://github.com/nodeca/js-yaml/commit/7b256d7612a4c29cdf5dfd48a15e01c8dbbb5291).

**Comment by kitos9112** [comment]
The extension itself does the job flawlessly, however, I cannot widely use it as much as I wanted due to the vast majority of my YAML manifests rely on comments quite heavily! 

Having [7b256d7 ](https://github.com/nodeca/js-yaml/commit/7b256d7612a4c29cdf5dfd48a15e01c8dbbb5291) implemented as part of v4.0.0 would definitely boost this feature 👍🏼
